### PR TITLE
first branch commit

### PR DESCRIPTION
**Pull request by bharadwajmolugu**

### CODE REVIEW

File: bharadwaj.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def sub(a,b):
+    return a-b
```
Comment (ArunKumar237) [inline bharadwaj.py]: pls call the function sub

Comment (bharadwajmolugu) [reply bharadwaj.py]: ok understood

Comment (bharadwajmolugu) [reply bharadwaj.py]: done I have called the function

Comment (ArunKumar237) [reply bharadwaj.py]: very good

**Pull request merged**